### ISSUE TITLE
Should natruralizing datasets unpack Overlay data/ RedPaletteColorLookupTableData, etc.

### ISSUE THREAD

**Issue by JamesAPetts** [dcmjs-org/dcmjs#114]
RE: https://github.com/OHIF/Viewers/pull/1546

If a tag is a bitpacked array, or an inline binary, should it be the naturalizer's responsibility to unpack/decode this?

seems more _natural_.

Obviously if data references a `BulkDataURI` it should not be dcmjs' responsibility to fetch this.

**Comment by pieper** [comment]
Yes, makes sense.  Can you add a link to an example dataset (maybe put it on dcmjs-org/data).
